raiwidgets/raiwidgets/explanation_dashboard.py

```diff
@@ -45,7 +45,6 @@ class and for the regression case a method of predict()
     def __init__(self, explanation, model=None, dataset=None,
                  true_y=None, classes=None, features=None,
                  public_ip=None, port=None, locale=None):
-        """Initialize the fairness Dashboard."""
```

Comment:
are you sure we should remove such comments?  maybe just update to say explanation dashboard?

Reply:
Do they add anything? The method is called `__init__` in a class called `ExplanationDashboard`. I'll add it back, but I'm typically in favor of useful comments only.